eng/pipelines/coreclr/exploratory.yml

```diff
@@ -0,0 +1,51 @@
+# This job definition automates the exploratory tool.
+
+trigger: none
+
+schedules:
+- cron: "0 6 * * *"
+  displayName: Daily at 11:30 PM (UTC-8:00)
```

Comment:
It seems like daily is too often, and weekly might make more sense. But I'm ok with this; we can change our mind about scheduling later.

Reply:
Ok, I will schedule it for weekly for now.